### PR TITLE
Minor typos, add docs

### PR DESCRIPTION
**Pull request by andrewkmin**
This PR intends to clean up some minor typos, as well as include some debugging tips re: setting up Rust

### CODE REVIEW

File: Solana_And_Web3/en/Section_2/Resources/m1_setup.md [hunk 1]
```diff
@@ -79,6 +79,17 @@ If you receieve an error that looks like this - `greadlink: command not found` y
...
+You may encounter additional errors like the following:
+```
+error: failed to download from `https://crates.io/api/v1/crates/console/0.11.3/download`
+
+Caused by:
+  [55] Failed sending data to the peer (Connection died, tried 5 times before giving up)
+```
+
+This is most likely an intermittent error; attempt to run the script again, and it should eventually download. If it _still_ doesn't end up downloading, you can try (locking your cargo version)[https://github.community/t/failed-sending-data-to-the-peer-connection-died-tried-5-times-before-giving-up/189130/4] and running it again.
+
```
Comment (adilanchian) [inline Solana_And_Web3/en/Section_2/Resources/m1_setup.md:92]: Hey a few things here - 

1. This line `If you receieve an error that looks like this - `greadlink: command not found` you will need to do two things:` there are three steps here now if you could update :)

2. This `(locking your cargo version)[https://github.community/t/failed-sending-data-to-the-peer-connection-died-tried-5-times-before-giving-up/189130/4]` is actually not written correctly, it should be: 
[locking your cargo version](https://github.community/t/failed-sending-data-to-the-peer-connection-died-tried-5-times-before-giving-up/189130/4)

File: Solana_And_Web3/en/Section_3/Lesson_2_Call_Deployed_Program.md [hunk 2]
```diff
@@ -274,7 +274,7 @@ Then, we just need to change up `renderConnectedContainer` to account for two ca
 
 ```jsx
 const renderConnectedContainer = () => {
-	// If we hit this, it means the program account hasn't be initialized.
+	// If we hit this, it means the program account hasn't been initialized.
```
Comment (adilanchian) [inline Solana_And_Web3/en/Section_3/Lesson_2_Call_Deployed_Program.md:277]: Can you remove these indents here please?

**Review by adilanchian — CHANGES_REQUESTED** [review]
Hey - 

Thanks for the changes here! Just a few thoughts here :)

**Comment by adilanchian** [comment]
Hey @andrewkmin !

Just throwing a quick tag here :). Let me know if you need help with any of these comments

File: Solana_And_Web3/en/Section_3/Lesson_2_Call_Deployed_Program.md (hunk 2 above)
Comment (andrewkmin) [reply Solana_And_Web3/en/Section_3/Lesson_2_Call_Deployed_Program.md]: done, let me know if this is what you intended to see :)

File: Solana_And_Web3/en/Section_4/Lesson_1_Finishing_Touches_Web_App_And_Program.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ It would be super cool if you loved a certain GIF submitted by another user SO M
 
 **Solana's super low gas fee means sending small amounts of money like this actually makes sense.** If you do this, you could even make a version of Patreon or BuyMeACoffee on Solana. Not that crazy. You have all the basic skills now.
 
-Who need Stripe and PayPal when you have a super low-gas fee blockchain that lets you instant payments?!?
+Who needs Stripe and PayPal when you have a super low-gas fee blockchain that lets you make instant payments?!?
```
Comment (andrewkmin) [inline Solana_And_Web3/en/Section_4/Lesson_1_Finishing_Touches_Web_App_And_Program.md:48]: let me know what you think about this change; i'm assuming we want to highlight the low-gas nature of solana, but could be wrong here 😄

**Comment by adilanchian** [comment]
Looks great! Thanks for working with me on this :)

**Comment by adilanchian** [comment]
ALSO - CONGRATS ON YOUR FIRST CONTRIBUTION

👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏

**Comment by adilanchian** [comment]
@all-contributors please add @andrewkmin for doc

**Pull request merged**